### PR TITLE
remove man 8 file, fix spec warnings

### PR DESCRIPTION
**Pull request by digitaltom**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/scc-connect-prophet/23165/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 0972845644169d58da42b143422276a74168fa85 into 399ef48bd77c2074b2fda08f8eaf3f37bc6efb60 )

File: package/SUSEConnect.spec [hunk 1]
```diff
@@ -66,8 +66,8 @@ install -D -m 644 %_sourcedir/SUSEConnect.5.gz %{buildroot}%_mandir/man5/SUSECon
 install -D -m 644 %_sourcedir/SUSEConnect.8.gz %{buildroot}%_mandir/man8/SUSEConnect.8.gz
 install -D -m 644 %_sourcedir/SUSEConnect.example %{buildroot}%_sysconfdir/SUSEConnect.example
 
-ln -s SUSEConnect.5.gz %{buildroot}%_mandir/man8/SUSEConnect-%{version}.5.gz
-ln -s SUSEConnect.8.gz %{buildroot}%_mandir/man8/SUSEConnect-%{version}.8.gz
+ln -s SUSEConnect.5.gz %{buildroot}%_mandir/man5/SUSEConnect.%{rb_default_ruby_suffix}.5.gz
```
Comment (kirushik) [inline package/SUSEConnect.spec]: :clap:

**Comment by kirushik** [comment]
LGTM

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
http://river.suse.de/job/scc-connect-prophet/23169/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged ec7024f98540cc727fad8d1c1ffa7d329283afa8 into 399ef48bd77c2074b2fda08f8eaf3f37bc6efb60 )

**Pull request merged**